radionets/dl_framework/model.py

```diff
@@ -401,3 +397,20 @@ def _conv_block(self, ni, nf, stride):
             nn.Conv2d(nf, nf, 3, stride=1, padding=1, bias=False),
             nn.BatchNorm2d(nf),
         )
+
+
+def even_better_symmetry(x):
+    upper_half = x[:, :, 0 : x.shape[2] // 2, :].clone()
+    upper_left = upper_half[:, :, :, 0 : upper_half.shape[3] // 2].clone()
+    upper_right = upper_half[:, :, :, upper_half.shape[3] // 2 :].clone()
+    a = torch.flip(upper_left, dims=[-2, -1])
+    b = torch.flip(upper_right, dims=[-2, -1])
+
+    upper_half[:, :, :, 0 : upper_half.shape[3] // 2] = b
+    upper_half[:, :, :, upper_half.shape[3] // 2 :] = a
+
+    # a = torch.zeros((x.shape))
```

Comment:
Same as above. Can uncommented rows be removed?

Reply:
Deleted